_posts/bad-at-to-do-lists-now-what.md

```diff
@@ -0,0 +1,65 @@
+- The calendar on the computer and phone
+- Phone calls and messages, including text messages, Signal, and WhatsApp. All groups, without exceptions, get muted on day one.
+Nothing else. And when it comes to phone calls and messages, my iPhone turns on "Do Not Disturb" automatically from 6 PM to  8 AM, in which case only 6 or 7 numbers can dial in. That is it.
+
+Running on zero-notifications is a relief; with no pressure coming from badges or any other sort of distracting and pressure-designed crap, my day is already doing great.
+
+However, as already said in the above-linked article, the day in Particular Software is a bit particular (no pun intended). Time zones, multiple task forces, wearing more than one hat ranging from "product management" to support, all participate in the risk of making the day a bit of a mess. 
+
+## Please welcome context switching 
+
+One of the side effects of wearing multiple hats throughout the day is the rise of context switching. Let's do some extreme math to put the issue in context; if we were to have only 5-minute long tasks and an 8-hour workday, the hypothetical result is 96 completed tasks: 12 tasks every hour, times 8 hours. Now, factor in multiple hats, and again for the sake of the sample, let's imagine that each assignment requires a different hat and that to change it (context switch), it takes our brain about 5 minutes. In such a scenario, we'll be able to complete 50% of the 96 tasks a day—a colossal cutoff.
+
+One could argue a zero-context switching approach would be an ideal world. Probably. I guess it depends on the person and the task. There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly; after a couple of hours in the best case, my mind starts digressing, signaling it's time for a change.
+
+> If there is an unsolved problem, like a bug or a not yet wholly identified solution, that's another story.
+
+I also enjoy many diverse activities ranging from solving technical problems to "product management" to more business-related activities,
+
+However, if I follow my brain's hint and switch to another activity, the effort required is extreme. In retrospect, I can state that the first 15 to 20 minutes of the next task are "wasted" cycles in the attempt to clear up my mind and get the necessary concentration to attack the new problem.
+
+> On a side note, that is the reason why I love "commuting" on foot. My office is precisely a mile from home. In the best case, it's an 8-minute walk (I'm in a hurry sometimes) or a 15-minute relaxing one. Either one is the perfect decompression time. By the time I get home, my brain is ready for the 3-year old little Dracula ;-)
+
+## Fillers
+
+Since a couple of months, so it's probably too early for some conclusions, I started injecting filler activities between daily tasks. One of the things I enjoy is reading.
+
+Reading turned out to be an excellent filler activity. Building LEGO is another satisfactory one.
```

Comment:
If you accept my second suggestion above, combine this sentence with the previous paragraph:

```suggestion
For me, reading is an excellent filler activity. Building LEGO is another satisfactory one.
```